pkg/server/plugin/upstreamauthority/spire/spire.go

```diff
@@ -340,23 +354,17 @@ func arePublicKeysEqual(a, b []*plugintypes.JWTKey) bool {
 	return true
 }
 
-func parseBundle(b *types.Bundle) *plugintypes.Bundle {
+func parseBundle(b *types.Bundle) (*plugintypes.Bundle, error) {
 	typesBundle := proto.Clone(b).(*types.Bundle)
```

Comment:
probably don't need to clone anymore since we're converting between types instead of holding onto a clone of the original